src/js/libs/ol.js

```diff
@@ -77,6 +77,7 @@ export default {
   },
 
   geom: {
+    LineString,
```

Comment:
I can't see where this module is used ?

Reply:
Here: 

https://github.com/c2corg/c2c_ui/blob/283c7f575de29a426446e31f83ee31a70f85f2df/src/views/portals/YetiView.vue#L952

Basically, it draws linestrings instead of polygon, to improve performance. See https://github.com/openlayers/openlayers/issues/10139

I notice that @ahocevar just replied to use `ol/layer/VectorImage` instead, but it's only v6. I will add `renderMode: 'image'` to increase perf (even if linestrings are still needed)